packages/core-accountsjs/core-accountsjs.js

```diff
@@ -0,0 +1,203 @@
+import { AccountsServer, ServerHooks, AccountsJsError } from '@accounts/server';
+import { AccountsPassword } from '@accounts/password';
+import MongoDBInterface from '@accounts/mongo';
+import { MongoInternals } from 'meteor/mongo';
+import { DatabaseManager } from '@accounts/database-manager';
+import { Random } from 'meteor/random';
+import crypto, { randomBytes } from 'crypto';
+import pick from 'lodash.pick';
```

Comment:
I'm opposed to adding any dependencies that are not needed for important reasons, check out https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_pick and I think you don't need to use defer either in a strictly node.js environment (https://medium.com/javascript-in-plain-english/setimmediate-vs-process-nexttick-in-nodejs-6fbfbebd60a3)

Reply:
The Lodash part is already done. I'mma see what can be done now about deferring...

Reply:
The defer is part of the original implementation. See https://github.com/accounts-js/accounts/blob/master/packages/password/src/accounts-password.ts#L648

I tried testing things out locally and they work fine but still unsure about the change so I decided to ask for help [here](https://spectrum.chat/accounts-js/general/usage-of-lodash-defer-in-password-package~064218c6-707d-4576-9e4e-85dc16bcd386).